scipy/optimize/tests/test_optimize.py

```diff
@@ -533,7 +533,7 @@ def cost(x):
         return rng.random(1) * 1000  # never converged problem
 
     for imaxfev in [1, 10, 50]:
-        for method in ['Powell']:  # TODO: extend to more methods
+        for method in ['Powell', 'Nelder-Mead']:  # TODO: extend to more methods
```

Comment:
```suggestion
        # TODO: extend to more methods
        for method in ['Powell', 'Nelder-Mead']:
```

Reply:
This is the reason why the lint check is failing.